src/widget/weffect.cpp

```diff
@@ -44,12 +44,13 @@ void WEffect::effectUpdated() {
         EffectPointer pEffect = m_pEffectSlot->getEffect();
         if (pEffect) {
             const EffectManifest& manifest = pEffect->getManifest();
-            if (! manifest.shortName().isEmpty()) {
+            if (!manifest.shortName().isEmpty()) {
                 name = manifest.shortName();
+                description = manifest.name() + ": " + manifest.description();
```

Comment:
Should the `": "` be translatable? @sblaisot, that should be `" : "` in French, correct?